smoketest.bash

```diff
@@ -64,7 +63,28 @@ while getopts "hs:prGtAOVXc:bnk" opt; do
             DEPLOY_GRAFANA=false
             ;;
         t)
-            INCLUDE_SAMPLE_APPS=true
+            nextopt=${!OPTIND}
+            if [[ -n $nextopt && $nextopt != -* ]] ; then
+                OPTIND=$((OPTIND + 1))
```

Comment:
I think this can be done by adding `:` after option `t` as in:

```
getopts "hs:prGt:AOVXc:bnk" opt
```

Reply:
Unfortunately not, because adding `:` this way indicates that the flag has a required argument. The way I write it here allows for the argument to be optional.

Reply:
Oh right right! Thanks!